Classes/Views/SelectableCell.swift

```diff
@@ -47,6 +57,9 @@ class SelectableCell: UICollectionViewCell {
 
     override func layoutSubviews() {
         super.layoutSubviews()
+        accessibilityLabel = contentView.subviews
+            .flatMap { $0.accessibilityLabel }
+            .reduce("", { "\($0 ?? "").\n\($1)" })
```

Comment:
That would be a little restricting though - we can't access the `viewModel` in the `NotificationCell`, which I used to provide the type (Issue / PR / commit).